src/Datagrid/Pager.php

```diff
@@ -26,11 +26,6 @@
  */
 class Pager extends BasePager
 {
-    /**
-     * Use separator in CONCAT() function for correct determinate similar records.
-     */
-    public const CONCAT_SEPARATOR = '|';
```

Comment:
This is a BC-break.

You have to keep it, and update the phpdoc
```
/*
 * NEXT_MAJOR: Remove this constant.
 *
 * @deprecated since sonata-project/doctrine-orm-admin-bundle 3.23 and will be removed in version 4.0.
 */
```

Reply:
Isn't it for internal use only?

Reply:
It's a public property and there is no @internal ; so you cant be sure someone didnt used it